hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/S3KeyGenerator.java

```diff
@@ -81,7 +103,8 @@ public Void call() throws Exception {
     if (endpoint.length() > 0) {
       amazonS3ClientBuilder
           .withPathStyleAccessEnabled(true)
-          .withEndpointConfiguration(new EndpointConfiguration(endpoint, ""));
+          .withEndpointConfiguration(
+              new EndpointConfiguration(endpoint, "us-east-1"));
```

Comment:
Any reason for this change, setting to us-east-1?

Reply:
Just one reason: got an exception ;-) The AWS 4 signature handling failed in our code with having empty region. I tried to define it with environment variable but it was not enough.